scripts/exrrfs_run_gsidiag.sh

```diff
@@ -301,6 +304,49 @@ if [ "${DO_RADDA}" = "TRUE" ]; then
        cp ./satbias_pc.out ${COMOUT}/rrfs.${spinup_or_prod_rrfs}.${YYYYMMDDHH}_satbias_pc
      fi
   fi
+
+fi
+
+#------------------------------------------------------------------------
+# set up local dirs and run run radmon to generate radiance monitor data
+#------------------------------------------------------------------------
+
+if [ DO_RADMON ]; then
+   echo "Run EMC Radmon package to generate daily monitoring data for satellite"
+
+   envir=${envir:-prod}
+   REGIONAL_RR=${REGIONAL_RR:-1}
+
+   export TANKverf=${TANKverf:-$NWGES_BASEDIR/radmon}
+   export TANKverf_rad=$TANKverf/radmon.$PDY
+
+
+   if [ ! -d $TANKverf_rad ]; then
+       mkdir -p -m 775 $TANKverf_rad
+   fi
+
+
+   export RAD_AREA=${RAD_AREA:-rgn}
+   export CYCLE_INTERVAL=${CYCLE_INTERVAL:-1}
+   export DATA=${analworkdir_conv}/radmon
+   export TANKverf_radM1=${TANKverf_radM1:-${TANKverf}/radmon.${PDY}}
+
+   export GSI_MON_BIN=$EXECDIR
```

Comment:
@xyzemc, please change `$EXECDIR` to `$EXECdir`

Reply:
done